python-recipes/RAG/01_redisvl.ipynb

```diff
@@ -270,11 +268,19 @@
         "outputId": "a8430acc-2e6d-45fd-fc8b-601fbbd8289b"
       },
       "outputs": [
+        {
+          "name": "stderr",
```

Comment:
Hate to do this but I think we should try to ignore these kinds of warnings if possible. Maybe a simple one liner will do it near top of notebook?